source/app/service-providers/translation-provider.ts

```diff
@@ -88,6 +88,12 @@ export default class TranslationProvider {
    * @return {Promise} Resolves if everything worked out, rejects otherwise.
    */
   async init (): Promise<void> {
+    const autoUpdate: boolean = global.config.get('autoUpdate')
+    if (!autoUpdate) {
```

Comment:
I'm unsure if that should be here, since users will associate "auto update" with the application updates themselves, and not necessarily translation updates, so we need to find a good solution here. Plus, maybe we should refactor the `init` so that `updateTranslations()` can be called if the setting is true. Otherwise this check here would make other potential init-actions impossible which might be necessary in the future.

Reply:
We could rename it to updateCheck/checkForUpdates, which probably is more reasonable. Also we could just wrap all the init code in that if